### PR TITLE
fixed mistakes

### PR DESCRIPTION
**Pull request by aliaksandr-harunou**

### CODE REVIEW

File: Airport.js [hunk 1]
```diff
@@ -15,12 +15,8 @@ class Airport {
 
     getPassengerPlaneWithMaxPassengersCapacity() {
         let passengerPlanes = this.getPassengerPlanes();
+        passengerPlanes.sort((firstPlane, secondPlane) => firstPlane.getPassengersCapacity() > secondPlane.getPassengersCapacity() ? -1 : 1);
         let planeWithMaxCapacity = passengerPlanes[0];
-        for (let i = 0; i < passengerPlanes.length; i++) {
-            if (passengerPlanes[i].getPassengersCapacity() > planeWithMaxCapacity.getPassengersCapacity()) {
-                planeWithMaxCapacity = passengerPlanes[i];
-            }
-        }
```
Comment (VladSelikh) [inline Airport.js:23]: You don't need `planeWithMaxCapacity` variable here.

File: models/experimentalTypes.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-const experimentalTypes   = {
-    LIFTING_BODY: 'Lifting_body',
-    HYPERSONIC: 'Hypersonic',
-    HIGH_ALTITUDE: 'High_altitude',
-    VTOL: 'Vtol'
+const EXPERIMENTALTYPES   = {
```
Comment (VladSelikh) [inline models/experimentalTypes.js]: Please add `_` between words.

File: models/militaryTypes.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-const militaryTypes = {
-    FIGHTER: 'Fighter',
-    BOMBER: 'Bomber',
-    TRANSPORT: 'Transport'
+const MILITARYTYPES = {
```
Comment (VladSelikh) [inline models/militaryTypes.js]: Please add `_` between words.

File: models/classificationLevels.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
-const classificationLevels  = {
-    UNCLASSIFIED: 'Unclassified',
-    CONFIDENTIAL: 'Confidential',
-    SECRET: 'Secret',
-    TOP_SECRET: 'Top_secret'
+const CLASSIFICATIONLEVELS = {
```
Comment (VladSelikh) [inline models/classificationLevels.js]: Please add `_` between words.

File: Airport.js [hunk 5]
```diff
@@ -56,24 +52,14 @@ class Airport {
...
 
     getMilitaryPlanesWithType(militaryPlaneType) {
-        let militaryPlanesWithType = [];
         let militaryPlanes = this.getMilitaryPlanes();
-        for (let i = 0; i < militaryPlanes.length; i++) {
-            if (militaryPlanes[i].getMilitaryType() === militaryPlaneType) {
-                militaryPlanesWithType.push(militaryPlanes[i]);
-            }
-        }
+        let militaryPlanesWithType = militaryPlanes.filter(plane => plane.getMilitaryType() === militaryPlaneType);
```
Comment (VladSelikh) [inline Airport.js]: You don't need `militaryPlanesWithType` variable here.

File: Airport.js [hunk 6]
```diff
@@ -56,24 +52,14 @@ class Airport {
...
 
     getExperimentalPlanesWithClassificationLevel(classificationLevel) {
-        let experimentalPlanesWithClassificationLevel = [];
         let experimentalPlanes = this.getExperimentalPlanes();
-        for (let i = 0; i < experimentalPlanes.length; i++) {
-            if (experimentalPlanes[i].classificationLevel === classificationLevel) {
-                experimentalPlanesWithClassificationLevel.push(experimentalPlanes[i]);
-            }
-        }
+        let experimentalPlanesWithClassificationLevel = experimentalPlanes.filter(plane => plane.classificationLevel === classificationLevel);
```
Comment (VladSelikh) [inline Airport.js]: You don't need `militaryPlanesWithType` variable here.

File: Airport.js [hunk 7]
```diff
@@ -91,9 +77,9 @@ class Airport {
         return this.planes;
     }
 
-    static print(planes) {
-        return JSON.stringify(planes);
+    print(providedPlanes) {
```
Comment (VladSelikh) [inline Airport.js]: Please remove this method to a separate function OUT of the `Airport` class.

File: Planes/PassengerPlane.js [hunk 8]
```diff
@@ -1,19 +1,15 @@
-const Plane = require('./Plane');
+const {Plane} = require('./Plane');
 
 class PassengerPlane extends Plane {
 
-    constructor(model, maxSpeed, maxFlightDistance, maxLoadCapacity, passengersCapacity) {
-        super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
-        this._passengersCapacity = passengersCapacity;
+    constructor(obj) {
```
Comment (VladSelikh) [inline Planes/PassengerPlane.js]: Please give more meaningful name to a parameter that is passed to the `constructor`.

File: Planes/MilitaryPlane.js [hunk 9]
```diff
@@ -1,4 +1,4 @@
-const Plane = require('./Plane');
+const {Plane} = require('./Plane');
 
 class MilitaryPlane extends Plane {
```
Comment (VladSelikh) [inline Planes/MilitaryPlane.js]: Why didn't you add object as a parameter to a `constructor` in `MilitaryPlane`, `ExperimentalPlane`, `Plane` classes?

File: Planes/ExperimentalPlane.js [hunk 10]
```diff
@@ -1,4 +1,4 @@
-const Plane = require('./Plane');
```
Comment (VladSelikh) [inline Planes/ExperimentalPlane.js:1]: Why didn't you rename `Plane` folder in plane` as it was highlighted in the comments of the previous Pull Request?

Comment (aliaksandr-harunou) [reply Planes/ExperimentalPlane.js:1]: http://prntscr.com/225vyp8

File: Airport.js (hunk 1 above)
Comment (aliaksandr-harunou) [reply Airport.js:23]: fixed

File: models/experimentalTypes.js (hunk 2 above)
Comment (aliaksandr-harunou) [reply models/experimentalTypes.js]: fixed

File: models/militaryTypes.js (hunk 3 above)
Comment (aliaksandr-harunou) [reply models/militaryTypes.js]: fixed

File: Airport.js (hunk 5 above)
Comment (aliaksandr-harunou) [reply Airport.js]: fixed

File: Airport.js (hunk 6 above)
Comment (aliaksandr-harunou) [reply Airport.js]: fixed

File: Airport.js (hunk 7 above)
Comment (aliaksandr-harunou) [reply Airport.js]: moved to the separate folder "helpers"

File: Planes/PassengerPlane.js (hunk 8 above)
Comment (aliaksandr-harunou) [reply Planes/PassengerPlane.js]: fixed

File: Planes/MilitaryPlane.js (hunk 9 above)
Comment (aliaksandr-harunou) [reply Planes/MilitaryPlane.js]: objects were added

File: models/classificationLevels.js (hunk 4 above)
Comment (aliaksandr-harunou) [reply models/classificationLevels.js]: fixed

**Review by VladSelikh — APPROVED** [review]

**Pull request merged**